addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -141,7 +141,7 @@
                     <div t-if="is_sample" class="h5 o_ribbon_right bg-primary text-uppercase">Sample</div>
                     <input type="hidden" name="product-id" t-att-data-product-id="record.id"/>
                     <a class="o_carousel_product_img_link position-absolute mx-auto" t-att-href="record.website_url">
-                        <img class="card-img-top" loading="lazy" t-att-src="data['image_512']" t-att-alt="record.display_name"/>
+                        <img class="o_img_product_cover card-img-top rounded" loading="lazy" t-att-src="data['image_512']" t-att-alt="record.display_name"/>
```

Comment:
I understand that the result might be better, but this crops the images according to its ratio, which wasn't the case before. If users want to keep their images fully visible, they will no longer be able to do so.
| Excepted | This branch |
|--------|--------|
| <img width="1340" alt="Capture d’écran 2024-10-01 à 10 48 28" src="https://github.com/user-attachments/assets/528bbd5e-1e97-4e88-9e85-cfddd5fd8df2"> | <img width="1375" alt="Capture d’écran 2024-10-01 à 10 48 22" src="https://github.com/user-attachments/assets/7cd9abf2-1b24-4f93-8f59-6de2dd41e550"> |

I think that since this is a fix, we should keep the current behavior. But it could be improved later.